### ISSUE TITLE
dev-gap/ctbllib-1.2_p2-r6 fetch failed

### ISSUE THREAD

**Issue by elchiquics** [cschwan/sage-on-gentoo#723]
```
>>> Downloading 'http://gentoo.mirrors.tds.net/gentoo/distfiles/layout.conf'
--2022-10-29 11:36:09--  http://gentoo.mirrors.tds.net/gentoo/distfiles/layout.conf
Resolving gentoo.mirrors.tds.net... 216.165.129.134
Connecting to gentoo.mirrors.tds.net|216.165.129.134|:80... connected.
HTTP request sent, awaiting response... Read error (Connection reset by peer) in headers.
Retrying.

--2022-10-29 11:36:49--  (try: 2)  http://gentoo.mirrors.tds.net/gentoo/distfiles/layout.conf
Connecting to gentoo.mirrors.tds.net|216.165.129.134|:80... connected.
HTTP request sent, awaiting response... Read error (Connection reset by peer) in headers.
Retrying.

--2022-10-29 11:37:30--  (try: 3)  http://gentoo.mirrors.tds.net/gentoo/distfiles/layout.conf
Connecting to gentoo.mirrors.tds.net|216.165.129.134|:80... connected.
HTTP request sent, awaiting response... Read error (Connection reset by peer) in headers.
Giving up.

!!! Couldn't download '.layout.conf.gentoo.mirrors.tds.net'. Aborting.
>>> Downloading 'http://gentoo.mirrors.tds.net/gentoo/distfiles/ctbllib-1.2_p2.tar.bz2'
--2022-10-29 11:38:16--  http://gentoo.mirrors.tds.net/gentoo/distfiles/ctbllib-1.2_p2.tar.bz2
Resolving gentoo.mirrors.tds.net... 216.165.129.134
Connecting to gentoo.mirrors.tds.net|216.165.129.134|:80... connected.
HTTP request sent, awaiting response... Read error (Connection reset by peer) in headers.
Retrying.

--2022-10-29 11:38:56--  (try: 2)  http://gentoo.mirrors.tds.net/gentoo/distfiles/ctbllib-1.2_p2.tar.bz2
Connecting to gentoo.mirrors.tds.net|216.165.129.134|:80... connected.
HTTP request sent, awaiting response... Read error (Connection reset by peer) in headers.
Retrying.

--2022-10-29 11:39:36--  (try: 3)  http://gentoo.mirrors.tds.net/gentoo/distfiles/ctbllib-1.2_p2.tar.bz2
Connecting to gentoo.mirrors.tds.net|216.165.129.134|:80... connected.
HTTP request sent, awaiting response... Read error (Connection reset by peer) in headers.
Giving up.

>>> Downloading 'http://ftp.free.fr/mirrors/ftp.gentoo.org/distfiles/80/ctbllib-1.2_p2.tar.bz2'
--2022-10-29 11:40:15--  http://ftp.free.fr/mirrors/ftp.gentoo.org/distfiles/80/ctbllib-1.2_p2.tar.bz2
Resolving ftp.free.fr... 212.27.60.27, 2a01:e0c:1:1598::1
Connecting to ftp.free.fr|212.27.60.27|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2022-10-29 11:40:15 ERROR 404: Not Found.

>>> Downloading 'ftp://212.110.163.12/gentoo/distfiles/80/ctbllib-1.2_p2.tar.bz2'
--2022-10-29 11:40:15--  ftp://212.110.163.12/gentoo/distfiles/80/ctbllib-1.2_p2.tar.bz2
           => ‘/var/cache/distfiles/ctbllib-1.2_p2.tar.bz2.__download__’
Connecting to 212.110.163.12:21... connected.
Logging in as anonymous ... Logged in!
==> SYST ... done.    ==> PWD ... done.
==> TYPE I ... done.  ==> CWD (1) /gentoo/distfiles/80 ... done.
==> SIZE ctbllib-1.2_p2.tar.bz2 ... done.

==> PASV ... done.    ==> RETR ctbllib-1.2_p2.tar.bz2 ... 
No such file ‘ctbllib-1.2_p2.tar.bz2’.

>>> Downloading 'https://mirrors.ircam.fr/pub/gentoo-distfiles/distfiles/80/ctbllib-1.2_p2.tar.bz2'
--2022-10-29 11:40:17--  https://mirrors.ircam.fr/pub/gentoo-distfiles/distfiles/80/ctbllib-1.2_p2.tar.bz2
Resolving mirrors.ircam.fr... 129.102.1.37, 2001:660:3004:4003::37:80
Connecting to mirrors.ircam.fr|129.102.1.37|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2022-10-29 11:40:18 ERROR 404: Not Found.

>>> Downloading 'http://ftp.belnet.be/pub/rsync.gentoo.org/gentoo/distfiles/80/ctbllib-1.2_p2.tar.bz2'
--2022-10-29 11:40:18--  http://ftp.belnet.be/pub/rsync.gentoo.org/gentoo/distfiles/80/ctbllib-1.2_p2.tar.bz2
Resolving ftp.belnet.be... 193.190.198.27, 2001:6a8:3c80::27
Connecting to ftp.belnet.be|193.190.198.27|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2022-10-29 11:40:18 ERROR 404: Not Found.

>>> Downloading 'http://gentoo.mirrors.ovh.net/gentoo-distfiles/distfiles/80/ctbllib-1.2_p2.tar.bz2'
--2022-10-29 11:40:18--  http://gentoo.mirrors.ovh.net/gentoo-distfiles/distfiles/80/ctbllib-1.2_p2.tar.bz2
Resolving gentoo.mirrors.ovh.net... 213.32.5.7
Connecting to gentoo.mirrors.ovh.net|213.32.5.7|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2022-10-29 11:40:18 ERROR 404: Not Found.

>>> Downloading 'https://files.gap-system.org/gap4/tar.bz2/packages/ctbllib-1r2p2.tar.bz2'
--2022-10-29 11:40:18--  https://files.gap-system.org/gap4/tar.bz2/packages/ctbllib-1r2p2.tar.bz2
Resolving files.gap-system.org... 131.246.120.130, 2001:638:208:ef61:0:ff:fe00:2
Connecting to files.gap-system.org|131.246.120.130|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2022-10-29 11:40:19 ERROR 404: Not Found.

!!! Couldn't download 'ctbllib-1.2_p2.tar.bz2'. Aborting.
```

**Comment by kiwifb** [comment]
Hum, it looks like I removed it from my mirrors when I should not have. Did you find a copy?

**Comment by kiwifb** [comment]
In any case, it is mirrored again.

**Comment by elchiquics** [comment]
I closed this issue because I think that was my mirrors, i have errors with other packages than aren't in this repo. Later, I removed the mirror that was causing error with this packages. Still not download correctly this package

```
>>> Downloading 'http://ftp.free.fr/mirrors/ftp.gentoo.org/distfiles/80/ctbllib-1.2_p2.tar.bz2'
--2022-10-29 21:12:20--  http://ftp.free.fr/mirrors/ftp.gentoo.org/distfiles/80/ctbllib-1.2_p2.tar.bz2
Resolving ftp.free.fr... 212.27.60.27, 2a01:e0c:1:1598::1
Connecting to ftp.free.fr|212.27.60.27|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2022-10-29 21:12:20 ERROR 404: Not Found.

>>> Downloading 'http://mirror.leaseweb.com/gentoo/distfiles/80/ctbllib-1.2_p2.tar.bz2'
--2022-10-29 21:12:20--  http://mirror.leaseweb.com/gentoo/distfiles/80/ctbllib-1.2_p2.tar.bz2
Resolving mirror.leaseweb.com... 23.82.143.211
Connecting to mirror.leaseweb.com|23.82.143.211|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2022-10-29 21:12:21 ERROR 404: Not Found.

>>> Downloading 'https://mirror.bytemark.co.uk/gentoo/distfiles/80/ctbllib-1.2_p2.tar.bz2'
--2022-10-29 21:12:21--  https://mirror.bytemark.co.uk/gentoo/distfiles/80/ctbllib-1.2_p2.tar.bz2
Resolving mirror.bytemark.co.uk... 212.110.163.12, 80.68.83.150, 2001:41c8:20:5e6::150, ...
Connecting to mirror.bytemark.co.uk|212.110.163.12|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2022-10-29 21:12:21 ERROR 404: Not Found.

>>> Downloading 'https://mirrors.ircam.fr/pub/gentoo-distfiles/distfiles/80/ctbllib-1.2_p2.tar.bz2'
--2022-10-29 21:12:21--  https://mirrors.ircam.fr/pub/gentoo-distfiles/distfiles/80/ctbllib-1.2_p2.tar.bz2
Resolving mirrors.ircam.fr... 129.102.1.37, 2001:660:3004:4003::37:80
Connecting to mirrors.ircam.fr|129.102.1.37|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2022-10-29 21:12:22 ERROR 404: Not Found.

>>> Downloading 'https://files.gap-system.org/gap4/tar.bz2/packages/ctbllib-1r2p2.tar.bz2'
--2022-10-29 21:12:22--  https://files.gap-system.org/gap4/tar.bz2/packages/ctbllib-1r2p2.tar.bz2
Resolving files.gap-system.org... 131.246.120.130, 2001:638:208:ef61:0:ff:fe00:2
Connecting to files.gap-system.org|131.246.120.130|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2022-10-29 21:12:23 ERROR 404: Not Found.

!!! Couldn't download 'ctbllib-1.2_p2.tar.bz2'. Aborting.
```

**Comment by kiwifb** [comment]
I just checked the ebuild and I need to point it to the right mirror. The gap-system location has a limited shelf life, which is why I stopped using it for every other gap packages. I had not noticed that that one was still pointing there.

**Comment by kiwifb** [comment]
should be fixed by af419e8c07fced1e10b44080bc63bf267e52d4e0 please check.

**Comment by elchiquics** [comment]
It's solved, thank you!
